views/overlays/scores.hbs

```diff
@@ -65,4 +78,21 @@
           document.getElementById('rightCharacter').className = null;
         }
 	});
+
+	var twitchSocket = io('/twitch');
+
+	twitchSocket.on('send twitch data', function(data){
+		if(data.twitchUsername) {
+        var ifrm = document.createElement('iframe'); 
+        ifrm.setAttribute('src', 'http://www.twitch.tv/'+data.twitchUsername+'/chat?popout='); 
+        ifrm.style.width = '320px'; 
+        ifrm.style.height = '800px'; 
+        document.getElementById('twitch-chat-placeholder').appendChild(ifrm); 
```

Comment:
Is this going to make multiple chat iframes if you happen to change stream or call `send twitch data`?